### PR TITLE
feat: add option mixinsDir for postcss-mixins

### PR DESCRIPTION
**Pull request by nlfonseca**

### CODE REVIEW

**Comment by satazor** [comment]
I don’t like having importPath and mixinsDir (path vs dir)

I would prefer to settle on either importPath & mixinsPath or importDir and mixinsDir.

**Comment by satazor** [comment]
@andrefgneves @antoniocapelo @andregoncalvesdev  what’s your opinion?

**Comment by vasco-santos** [comment]
I agree with @satazor . I think that we should go with **path** for both.

**Comment by andregoncalvesdev** [comment]
i would go with `importPath & mixinsPath`

**Comment by nlfonseca** [comment]
I'm using ```mixinsDir``` because is the name of the property that the plugin expect.
A user that already knows this plugin should be familiar with the name.

I'm fine changing the name to ```mixinsPath``` but I felt the need to point this out.

what's your opinion?

@satazor @antoniocapelo @andregoncalvesdev @vasco-santos

**Comment by antoniocapelo** [comment]
@satazor I'm fine either way (using `path` or `dir` on both of the properties).
The upside of using `path` is that we're in fact passing a path, not a dir...
The upside of using `dir` is that is more compliant with the `postcss-mixins` plugin...

Let's flip a coin :P 

(in the end, `path` is more correct, I guess)

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ If you are developing a project that uses new CSS language features and must wor
 - You can use any features from [cssnext](http://cssnext.io/)
 - Enables [postcss-import](https://github.com/postcss/postcss-import) so that `@import` statements are inlined
 - Enables [postcss-mixins](https://github.com/postcss/postcss-mixins) so that you can define mixins
+- Enables [postcss-conditionals](https://github.com/andyjansson/postcss-conditionals) so that you can define @if statements
```
Comment (satazor) [inline README.md]: Could you put @if under backticks? `@if`

File: README.md [hunk 2]
```diff
@@ -49,6 +50,7 @@ module.exports = require('postcss-preset-moxy')();
 ```js
 module.exports = require('postcss-preset-moxy')({
     importPath: './src/styles',
+    mixinsPath: './src/styles/imports/mixins',
```
Comment (satazor) [inline README.md]: Could you change this to `./src/styles/mixins`?

File: README.md (hunk 1 above)
Comment (nlfonseca) [reply README.md]: done :)

**Pull request merged**